add package-asset docs (#1484)
cocos-creator/creator-docs: en/editor/assets/index.md

- Shortcut key `Esc` to cancel rename
- In addition, the initial name of the Typescript asset will be treated as its `className`, and the `className` cannot be repeated.

### Exporting Assets Package

> Select the resource to be exported, right-click on it, then select **Export Assets Package**. The **Export Assets** panel will automatically list the currently selected resource and its dependencies. To not export the dependent resources, uncheck **Include Dependency** in the bottom left corner of the **Export** panel.

Reviewer: ```suggestion
Select the asset to be exported, right-click on it, then select **Exporting Assets Package**. The **Exporting Assets** panel will automatically list the currently selected asset and its dependencies. To not export the dependent assets, uncheck **Include Dependency** in the bottom left corner of the **Exporting Assets** panel.
```

> Select the resource to be exported and right click on it, then select **Export Resource Package**. The **Export Resource** panel will automatically list the currently selected resource and its dependencies. If you don't need to export dependent resources, you can uncheck **Include Dependent Resources** in the bottom left corner of the **Export Resources** panel.

Reviewer 2: Select the resource to be exported and right click on it,
-->
Select the resource to be exported, right-click on it,

Reviewer 2: If you don't need to export dependent resources, you can uncheck **Include Dependent Resources** in the bottom left corner of the **Export Resources** panel.
-->
To not export the dependent resources, uncheck **Include Dependent Resources** in the bottom left corner of the **Export Resources** panel.

> Once identified, the resources can be exported by clicking the **Export** button to bring up the file storage dialog box where it is necessary to specify a folder location and file name.

Reviewer: ```suggestion
Once identified, the assets can be exported by clicking the **Export** button to bring up the file storage dialog box where it is necessary to specify a folder location and file name.
```

> Once you have identified the resources to be exported, click the **Export** button to bring up the file storage dialog box where you will need to specify a folder location and file name.

Reviewer 2: Once you have identified the resources to be exported, click the **Export** button to bring up the file storage dialog box where you will need to specify a folder location and file name.
-->
Once identified, the resources can be exported by clicking the **Export** button to bring up the file storage dialog box where it is necessary to specify a folder location and file name.

![exporting](img/package-asset/exporting.png)

> > **Note**: Exported zip packages are only suitable for **importing resource packages** use, as described below.

Reviewer 2: Exported
-->
exported

### Importing Assets Package

> Select and right click on the folder to import resources from, or right click on the **blank space** in the **Assets** panel, then select **Importing Assets Package** and select the zip package exported from **Exporting Assets Package** in the file browse dialog box. The resources in the zip package will be automatically parsed into the **Importing Assets Package** panel that pops up.

Reviewer: ```suggestion
Select and right click on the folder to import assets from, or right click on the **blank space** in the **Assets** panel, then select **Importing Assets Package** and select the zip package exported from **Exporting Assets Package** in the file browse dialog box. The assets in the zip package will be automatically parsed into the **Importing Assets Package** panel that pops up.
```

> Select and right click on the folder you want to import resources from, or right click on the **blank space** in the **Export Explorer** panel, then select **Import Package** and select the zip package exported from **Export Package** in the file browse dialog box. The resources in the zip package will be automatically parsed into the **Import Package** panel that pops up.

Reviewer 2: you want to import resources
-->
to import resources

> The import process will also ask to reconfirm the imported resources. Uncheck resources to remove the unnecessary ones.

Reviewer: ```suggestion
The import process will also ask to reconfirm the imported assets. Uncheck assets to remove the unnecessary ones.
```

> The import process will also ask the user to reconfirm the imported resources, so you can uncheck some resources to remove the ones you don't want to import.

Reviewer 2: The import process will also ask the user to reconfirm the imported resources, so you can uncheck some resources to remove the ones you don't want to import.
-->
The import process will also ask to reconfirm the imported resources. Uncheck resources to remove the unnecessary ones.

![importing](img/package-asset/importing.png)

> > **Note**:

Reviewer 2: **Note**:
-->
**Notes**:

> > 1. Imported zip packages, only zip packages exported by Cocos Creator and above using **Export Resource Packages** are supported.

Reviewer 2: Imported zip packages, only zip packages exported by Cocos Creator and above using **Export Resource Packages** are supported.

I don't quite understand what the meaning of this is.

Reviewer: The imported asset packages are only supported for zip packages exported by Cocos Creator 3.0 and above using the **Exporting Assets Package** feature.

Reviewer 2: that's perfect @xunyi0

> > 1. Imported zip packages, only zip packages exported by Cocos Creator 3.0.0 and above using **Export Resource Packages** are supported.

Reviewer 2: Cocos Creator 3.0.0
-->
Creator

We dont need to specify v3 since we are in the v3 docs when reading this.
